bluesky/plans.py

```diff
@@ -1331,7 +1331,66 @@ def rel_spiral(detectors, x_motor, y_motor, x_range, y_range, dr, nth,
                               x_range, y_range, dr, nth, tilt=tilt,
                               per_step=per_step, md=_md))
 
+def spiral_square(detectors, x_motor, y_motor, x_centre, y_centre, x_range,
```

Comment:
See my comment above. Covered by #1001, should be removed.

Reply:
This is a github error that  I will try to rectify now. for some reason it added the commits from the square_spiral branch along with my new ones.